lib/tasks/evm_dbsync.rake

```diff
@@ -74,7 +74,7 @@ namespace :evm do
       unless (total = RrPendingChange.count) == 0
         require 'progressbar'
         require 'rubyrep'
-        pbar = ProgressBar.new("Backlog", total)
+        pbar = ProgressBar.create(:title => "Backlog", :total => total, :autofinish => false)
```

Comment:
looks like `ProgressBar.create` delegates the args to another class, so it looks like `ProgressBar.new` would generate an argument error